dod-wrapper/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
manifest shouldn't be required for this, it's only for strictly android libs like scheduler

Reply:
Done 😁